cypress/components/contactuscomponent.cy.tsx

```diff
@@ -0,0 +1,65 @@
+
+import ContactForm from "../../src/components/molecules/contact-form/contact-form"
+
+describe('Checks the Contact Us component for accepting three cases of inputs', () => {
+  beforeEach(()=> {
+    cy.mount(<ContactForm />)
+    // input text field for name
+    cy.get('[data-cy="name-input"]').as('nameInputTextField')
+            
+    // input text element for email
+    cy.get('[data-cy="email-input"]').as('emailInputTextField')
+    'data-cy="note-input"'
+
+    // input element for message 
+    cy.get('[data-cy="note-input"]').as('messageEmailTextArea')
+
+    // button element 
+    cy.get('[data-cy="send-msg-btn"]').as('sendMessageBtn')
+
+  })
+
+    it('Case 1: Show error messages in a new span element when no text is provided in name, email and message inputs in the component', ()=> {
+        cy.get('@nameInputTextField').clear()
+        cy.get('@emailInputTextField').clear()
+        cy.get('@messageEmailTextArea').clear()
+
+        cy.get('@sendMessageBtn').click()
+
+        cy.get('[data-cy="name-error-msg"]').contains('Please add a name')
+        cy.get('[data-cy="email-error-msg"]').contains('Please add a proper email address')
+        cy.get('[data-cy="note-error-msg"]').contains('Please add a message')
+    })
+
+    it("Case 2 : Show the success message after proper inputs for name, email and message inputs are provided", () => {
+      cy.fixture('example').as('exampleInput')
+      cy.get('@nameInputTextField').type('Tausif Qureshi')
+      cy.get('@emailInputTextField').type('tausif2606@gmail.com')
+      cy.get('@messageEmailTextArea').type('Hello tech is hiring!')
+
+      cy.intercept('POST','/api/process-email', (req) => {
+        const statusCode = 200
+        req.reply(statusCode, {data:"Email sent"})
+
+      })
+      cy.get('@sendMessageBtn').click()
```

Comment:
<img width="516" alt="Screenshot 2023-05-30 at 6 46 53 PM" src="https://github.com/TechIsHiring/techishiring-website/assets/59664677/a4cff4f8-a7d2-4884-a932-2d678db245e5">

After running `npx cypress open --component` then in the browser test environment the above request made is shown to not be going to the actual server rather than is going to the stubbed request with a mock reply as shown in the above image. No emails are being sent to the given mail. I checked the [cypress docs](https://docs.cypress.io/guides/guides/network-requests#Command-Log) and it refers that a request that goes to the actual server is shown with a filled circle and the mocked requests endpoints are shown with the outlined circle as it is in the image.

Reply:
Ah okay, that makes sense. My bad for taking so long to get back to you.

Reply:
Could you work on the other issues that I pointed out in the review please?